apps/avifenc.c

```diff
@@ -890,7 +890,7 @@ int main(int argc, char * argv[])
         avifImageSetProfileICC(image, iccOverride.data, iccOverride.size);
     }
     if (exifOverride.size) {
-        avifImageSetMetadataExif(image, exifOverride.data, exifOverride.size);
+        avifImageSetMetadataExif(image, exifOverride.data, exifOverride.size, /*extractExifOrientationToIrotImir=*/AVIF_TRUE);
```

Comment:
Adding the extractExifOrientationToIrotImir parameter is an incompatible change. We can remain backward compatible by adding a new function to extract Exif orientation. We can just make `avifExtractExifOrientationToIrotImir()` a public API function.

Reply:
Done. I dislike having to rely on the user to remember to call `avifImageExtractExifOrientationToIrotImir()` though.